build.gradle

```diff
@@ -4,7 +4,7 @@ apply plugin: 'com.vanniktech.android.junit.jacoco'
 
 buildscript {
     ext.versions = [
-        testRules: '0.5',
+        testRules: '1.0.0',
```

Comment:
i haven't had luck with 1.0.0 and espresso 3.0.0. Did you test locally whether the screenshots are being taken or not. You can do so using `gw assembleDebug assembleAndroidTest && fastlane screengrab`